### DISCUSSION TITLE
Add AVIF format support

### DISCUSSION THREAD

**Discussion by nczz** [ddur/Warp-iMagick#10] [Ideas]
Hi! Thanks for your awesome plugin!

Do you have plan to add AVIF format support at the future version?

**Comment by ddur** [comment]
Yes, in the future, but ...

1) Compared to [WebP browser support](https://caniuse.com/webp) , [AVIF is still not well enough supported format](https://caniuse.com/avif).

2) Having WebP + another set of AVIF clones generated, may be to much disk space, for some users. So there will be probably a choice between WebP/AVIF format clones.

3) It is still unknown (to me) how to recognize AVIF compliant browsers in .htaccess, while `image/webp` in the HTTP_ACCEPT header is well established method to detect browsers supporting WebP images.

**Comment by ddur** [comment]
[PHP 8.1: GD: AVIF image support](https://php.watch/versions/8.1/gd-avif)
